doc/nrf/ug_nrf5340.rst

```diff
@@ -147,6 +152,9 @@ Most of these samples should run on the nRF5340 DK, but not all have been thorou
 
 Some Bluetooth LE samples require configuration adjustments to the :ref:`zephyr:bluetooth-hci-rpmsg-sample` sample as described in the `Network samples`_ section.
 
+The |NCS| also provides :ref:`Thread <ug_thread>` and :ref:`Zigbee <ug_zigbee>` samples that can be run on the nRF5340 DK.
```

Comment:
Do we need to say that the child image is built & flashed automatically.

Reply:
@grochu you were planning to rephrase the sentence at the start of the page. Can we maybe rework that sentence and this one in this PR?

Reply:
In the _Application Samples_ and _Network Samples_ sections wouldn't it be cleaner to only mention the specific applications available for these cores respectively, and only describe the automatic building of the child image in the _Building and Programming a sample_ section as it is stated for BLE?

Reply:
I mean also line 152 could be moved (and maybe reworded) to _"Building and programming a sample"_